### PR TITLE
Fix html5apptests

### PR DESCRIPTION
**Pull request by imalberto**
html5apptest_descriptor.json

Total Number of Executed Tests: 58
Total Number of Passed Tests: 46
Total Number of Failed Tests: 12
Total Number of Skipped Tests: 0

### CODE REVIEW

**Comment by imalberto** [comment]
Failing tests are related to composite execution on client. Might have been fixed with #742

**Comment by imalberto** [comment]
@caridy Sent you the list of failing tests

File: tests/func/applications/frameworkapp/common/application.json [hunk 1]
```diff
@@ -1,400 +1,406 @@
...
+  {
+    "settings": [ "master" ],
+    "config": {
+      "myconfig0": "This is myconfig0 from top application.json",
+      "myconfig1": "This is myconfig1 from top application.json",
+      "myconfig2": "This is myconfig2 from top application.json",
+      "myconfig3": "This is myconfig3 from top application.json"
+    },
+    "appPort": 4081,
+    "log": {
```
Comment (caridy) [inline tests/func/applications/frameworkapp/common/application.json]: this is not longer supported, use yui->config->debug, logLevel and filter accordingly.

File: tests/func/applications/frameworkapp/common/application.json [hunk 2]
```diff
@@ -1,400 +1,406 @@
...
+        "level": "debug",
+        "yui": false
+      },
+      "server": {
+        "level": "mojito",
+        "yui": false
+      }
+    },
+    "yui": {
+      "extraModules": ["plugin","jsonp"],
```
Comment (caridy) [inline tests/func/applications/frameworkapp/common/application.json]: this is not longer supported

File: tests/func/applications/frameworkapp/common/application.json [hunk 3]
```diff
@@ -1,400 +1,406 @@
...
+      "mojits_subdir1/*/",
+      "mojits_subdir2/*/",
+      "../test_files/mojits/"
+      ],
+      "mojitDirs": [
+        "just_one_mojit/one_level_in/Binders",
+        "mojits_params/*"
+        ],
+        "staticHandling": {
+          "useRollups": true,
```
Comment (caridy) [inline tests/func/applications/frameworkapp/common/application.json]: this is not longer supported.

File: lib/app/middleware/mojito-handler-static.js [hunk 4]
```diff
@@ -81,7 +81,14 @@ function modified(req, headers) {
  */
 function etag(data, stat) {
     // using data.length instead of stat.size to support compilation
-    return data.length + '-' + Number(stat.mtime);
```
Comment (caridy) [inline lib/app/middleware/mojito-handler-static.js:84]: This is not longer needed, was solved at the store level, files will always have `stat.mtime`.

**Comment by caridy** [comment]
+1

**Pull request merged**